spring-integration-amqp/src/main/java/org/springframework/integration/amqp/outbound/AsyncAmqpOutboundGateway.java

```diff
@@ -41,6 +44,8 @@
 
 	private final MessageConverter messageConverter;
 
+	private volatile boolean requiresReply; // overrides super
```

Comment:
Polishing for your consideration: https://github.com/artembilan/spring-integration/commit/a32c134a6b843f169d624473d9aeb36193698ad3